README.md

```diff
@@ -2,25 +2,36 @@
 
 Drought-related food insecurity in Chad, Somalia, Ethiopia and Malawi. 
 
+## Getting started
+If using Anaconda, set-up an environment and install the packages from `environment.yml`. 
+   ``` bash
+   conda env create --file environment.yml --name aafi
+   conda activate aafi
+   ```
+
 ### Computation for existing country
+FewsNet
 1. Run `convert_fewsnet_to_admin2.py` giving the country iso3 as arg
-2. Run `fewsnet_triggers.py` giving the country iso3 as arg
+2. Run `fewsnet_combined.py` giving the country iso3 as arg
```

Comment:
Why are there two assignments of country iso 3?

Reply:
Got the confusion, country iso 3 is input argument. Changed it now